VERSION

```diff
@@ -0,0 +1 @@
+0.13.0
```

Comment:
I'm still not fully convinced that this is the better approach as it is redundant to tags. But I'm Ok with it as we can also move back to using git.

Reply:
> I'm still not fully convinced that this is the better approach as it is redundant to tags

using tag in local doesn't work (see: https://github.com/ninech/netbox-client-ruby/issues/85)

also using tag in gemspec file is quite unusual in the Ruby community.

VERSION is also used here https://github.com/ninech/netbox-client-ruby/blob/master/bin/console#L20 which should have break some time ago.